src/main/java/org/wahlzeit_revisited/main/DatabaseMain.java

```diff
@@ -5,21 +5,14 @@
 import org.wahlzeit_revisited.utils.SysLog;
 import org.wahlzeit_revisited.utils.WahlzeitConfig;
 
-import java.sql.SQLException;
-
 public class DatabaseMain extends ModelMain {
 
     public DatabaseMain(WahlzeitConfig config) {
         super(config);
     }
 
     public void shutDown() {
-        try {
-            saveAll();
-            SessionManager.dropThreadLocalSession();
-            SysLog.logSysInfo("Shutting down database");
-        } catch (SQLException sqlException) {
-            SysLog.logThrowable(sqlException);
-        }
+        SessionManager.dropThreadLocalSession();
```

Comment:
do we still have sessions?

Reply:
No, but the global DatabaseConnection is still hold by this Singleton

renamed it: 5babe35